src/track/text-track.js

```diff
@@ -0,0 +1,15 @@
+//@flow
+import Track from './track'
+
+export default class textTrack extends Track {
+  _kind: string;
+
+  get kind(): string {
+    return this._kind;
+  }
+
+  constructor(settings: Object = {}) {
+    super(settings);
+    this._kind = settings.kind;
```

Comment:
In hls subtitles has an id. Why u decided to implement text track without id propery?